.github/workflows/release.yml

```diff
@@ -0,0 +1,123 @@
+name: Tinybird Release
+
+on:
+  workflow_call:
+    inputs:
+      job_to_run:
+        description: 'Select the job to run manually'
+        required: true
+        type: string
+        default: 'promote'
```

Comment:
it'd be nice to be able to parameterize the semver, specially to remove old releases

Reply:
True, it would be very useful for `rm`. We can allow override the one taking current `.tinyenv`.  

Let me do it on a separate MR as we need to change also on gitlab jobs and I don't want to mix it

Reply:
https://github.com/tinybirdco/ci/issues/30